test/blackbox-tests/test-cases/pkg/ocamlformat/ocamlformat-patch-extra-files.t

```diff
@@ -3,17 +3,21 @@ ocamlformat and its dependencies during the same run of the "dune fmt" command.
 loaded before the lock files are produced, this is why any 'patch' file inside
 'dev-tools.locks/ocmalformat' is not copied inside the 'build' directory when a rule depends on it.
 
-In the case of this issue, there is a rule that depends on an 'patch' file in order to copy the file
+The issue was that there is a rule that depends on an 'patch' file in order to copy the file
 inside '_private/default/..' directory, since the file could not be copied, the rule is not activated.
 So it fails when 'dune' trying to apply the patch. After any follwing run of 'dune fmt', it works
 because the 'patch' file is already present.
 
+The issue is now fixed.
```

Comment:
I don't think we should write this sort of comment in our tests. The test should describe the behaviour we're trying to preserve, but it's not necessary to describe the original problem or the fact that it was fixed. If anything, include a link to the github issue so people can read more about it there.